### PR TITLE
Some translates and fixes

### PR DESCRIPTION
**Pull request by 3dkrender**

### CODE REVIEW

File: docs/en/api-reference/rpc_api.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ curl --request POST \
 You can also call these endpoints using **cleos** commands:
 
 ```
-cleos -u https://chain.wax.io get info
+cleos -u https://apiwax.3dkrender.com get info
```
Comment (rakeden) [inline docs/en/api-reference/rpc_api.md:33]: sure you want to add your public API to the main docs?

**Review by rakeden — COMMENTED** [review]
Thanks for adding more translations!

File: docs/en/api-reference/rpc_api.md (hunk 1 above)
Comment (3dkrender) [reply docs/en/api-reference/rpc_api.md:33]: Well... it's an example. And it is public. Maybe something like
`cleos -u [api-url] get info` ?

Comment (rakeden) [reply docs/en/api-reference/rpc_api.md:33]: Looking good!

```suggestion
cleos -u [api-url] get info
```

**Pull request merged**